demo/demo.html

```diff
@@ -7,7 +7,7 @@
 	<base href=".."></base>
 
 	<script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.0.8/angular.min.js"></script>
-	<script src="http://code.jquery.com/jquery-1.10.1.min.js"></script>
+	<script src="https://code.jquery.com/jquery-1.10.1.min.js"></script>
```

Comment:
@adamhotep You asked a question, and I only answered it :) I'm not arguing with the statement that Paul Irish is making; I fully agree with it.